### PR TITLE
feat : Added landing page

### PR DESCRIPTION
**Pull request by aiman-ansari**
Added Landing page .
Added Images in public folder
Files to check

Components/Category.js
db/categories.js
Home.js
Context/DataContext
Preview Link
https://deploy-preview-4--shimmering-unicorn-9d8e94.netlify.app

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
 
-
+import Cart from "./pages/Cart/Cart";
 function App() {
   return (
     <div className="App">
+      
       <Header />
-        <Routers />
+      {/* <Home /> */}
```
Comment (muskanbagrecha) [inline src/App.js]: Remove commented code.

File: src/Components/Category/Category.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+export default function Category({item}){
+    return(
+                <div className="card box-shadow-bottom">
+                    <img className="img-sm" src={item.image} alt={item.categoryName}/>
```
Comment (muskanbagrecha) [inline src/Components/Category/Category.js:4]: You can destructure the properties of item.

File: src/Components/Product/Product.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+export default function Product({prod}){
+    return(
+                <div className="card box-shadow-bottom">
+                    <img className="img-sm" src={prod.image} alt={prod.title}/>
```
Comment (muskanbagrecha) [inline src/Components/Product/Product.js]: Can destructure properties of product.

File: src/Router.js [hunk 4]
```diff
@@ -4,14 +4,16 @@ import Cart from './pages/Cart/Cart'
 import Product from './pages/Product/Product'
 import Wishlist from './pages/Wishlist/Wishlist'
 import Login from './pages/Login/Login'
+import Mockman from 'mockman-js'
 export default function Routers(){
     return(
         <Routes>
-            <Route path="home" element={<Home />}/>
+            <Route path="/" element={<Home />}/>
```
Comment (muskanbagrecha) [inline src/Router.js:11]: Can add a fallback route using * for 404.

File: src/pages/Home/Home.js [hunk 5]
```diff
@@ -1,7 +1,38 @@
+import '../../App.css'
+import Category from "../../Components/Category/Category";
+import Product from '../../Components/Product/Product';
+import { categories } from '../../backend/db/categories';
+import { products } from '../../backend/db/products';
```
Comment (muskanbagrecha) [inline src/pages/Home/Home.js]: Do update this in future to fetch data using the api.

File: src/pages/Home/Home.js [hunk 6]
```diff
@@ -1,7 +1,38 @@
...
+            <div className="flex-align-center gap-2">
+                {categories.map((item)=>
+                    <Category item={item}/>
+                )}
+            </div>
+            <div className="flex-align-center m-2">
+                <h4>Top Deals</h4>
+            </div>
+            <div className="flex-align-center gap-2">
+                {products.map((prod)=>
```
Comment (muskanbagrecha) [inline src/pages/Home/Home.js]: Naming can be improved here.

**Review by muskanbagrecha — APPROVED** [review]
Good work overall. Do implement the suggested changes.

File: src/Router.js (hunk 4 above)
Comment (aiman-ansari) [reply src/Router.js:11]: I will do that later

File: src/App.css [hunk 7]
```diff
@@ -1,51 +1,22 @@
...
-  font-size: 3rem;
-  color: #d9d9d9;
-}
-
-.brand-title span {
-  color: #ffde59;
+  flex-direction: column;
+  padding: 1rem;
+  gap: 1rem;
+  background-color: rgb(232, 234, 235);
```
Comment (saurabhchirde) [inline src/App.css]: Color variables.

**Review by saurabhchirde — APPROVED** [review]
Clean and easy to read code. You can proceed to merge.

File: src/App.css (hunk 7 above)
Comment (aiman-ansari) [reply src/App.css]: Thank you for pointing it out

File: src/Components/Product/Product.js (hunk 3 above)
Comment (aiman-ansari) [reply src/Components/Product/Product.js]: sure I will do that later

**Pull request merged**